code/modules/mob/living/carbon/life.dm

```diff
@@ -339,12 +339,14 @@
 
 	return TRUE
 
-//Fourth and final link in a breath chain
+/// Fourth and final link in a breath chain
 /mob/living/carbon/proc/handle_breath_temperature(datum/gas_mixture/breath)
 	// The air you breathe out should match your body temperature
 	breath.temperature = bodytemperature
 
+/// Attempts to take a breath from the external or internal air tank.
 /mob/living/carbon/proc/get_breath_from_internal(volume_needed)
+<<<<<<< HEAD
 	if(internal)
 		if(internal.loc != src && !(wear_mask.clothing_flags & MASK_EXTEND_RANGE)) //SKYRAT EDIT ANESTHETIC MACHINE. ORIGNIAL CODE: if(internal.loc != src)
```

Comment:
I see the edit here looks for one more clothing flag. We can delete this section and that can be added to the checks in `invalid_internals()`